ROADMAP.md

```diff
@@ -0,0 +1,191 @@
+
+### 1. Testground provides reliable results
+
+**Why:** an unreliable testing platform is just a noise machine. We need to secure our users' trust. Testground maintainers need clear feedback about stability improvements & regressions.
+
+#### Milestone 1: We have a stability metrics dashboard
+
+Maintainers and users have a way to measure and follow Testground's Stability over any "relevant" axis.
+
+This might combine different languages, runners (k8s, docker, local), and context (developer env, CI env, k8s env).
+
+#### Milestone 2: We have identified and reached our user's stability requirements. We resolved "most" of them
+
+- local runner,
+- docker runner,
+- EKS runner,
+
+#### Milestone 3: We have a performances dashboard
+
+Maintainers and users have a way to measure and follow Testground's Performances over any "relevant" axis.
+
+### 2. Testground can be set up as an organization-wide Service
+
+**Why:** We believe Testground can provide value across entire organizations. Making it easy to run a large-scale workload and efficient small-scale CI tests are core to its success.
+
+#### Milestone 1: We can use Testground EKS clusters in production
+
+- Deploy
+- Security
+- Stability
+
+#### Milestone 2: We can measure and improve Testground's EKS stability
+
+- Deploying short-lived clusters for benchmarking
+
+#### Milestone 3: We can measure and improve Testground's EKS performances
+
+- Deploying short-lived clusters for benchmarking
+
+#### Milestone 4: Dashboard & CI Toolings
+
+### 3. The knowledge required to work efficiently with Testground is available and easy to access
+
+**Why:** We believe testing is valuable when everyone on the team can contribute. Our platform has to be approachable.
+
+Related - [EPICS 1741](https://github.com/testground/testground/issues/1471).
+
+#### Milestone 1: There is a working quickstart that is the entry point to every other documentation
+
+#### Milestone 2: Testground has an official release process
+
+- We distribute versions following an explicit release process (no more `:edge` by default)
+- It's easy to follow changes (CHANGELOG)
+- We distribute binaries
+
+#### Milestone 3: Testground provides extensive documentation
+
+- The documentation is up-to-date
+  - Generates configuration documentation
+- We provide introduction guides for every language
+- We provide doc for Common Patterns
+
+#### Milestone 4: Potential users know about Testground's existence and features
+
+- "Public Relations".
+
+### 4. Testground Development follows high-standard
+
+**Why:** Testground has already provided value amongst many different projects. Now, we need bulletproof development processes to make the project sustainable and facilitate external contributions.
+
+#### Testground CI & Testing is practical and reliable
```

Comment:
Are we dropping "milestones" pattern here on purpose?

Reply:
Yes, there are no strong dependencies between these tasks here.